apps/client/relay/trainings/CreateOrUpdateTrainingMutation.ts

```diff
@@ -1,17 +1,15 @@
 import { graphql } from 'react-relay';
 
 export const CreateOrUpdateTrainingMutation = graphql`
-mutation CreateOrUpdateTrainingMutation($_id: ID, $name: String!, $description: String!, $exercises: [ExerciseInput!]!) {
+mutation CreateOrUpdateTrainingMutation($_id: ID, $name: String!, $exercises: [ExerciseInput!]!) {
```

Comment:
prefer to use `mutation CreateOrUpdateTrainingMutation($input: CreateOrUpdateTrainingInput!)`